akka-docs/src/main/paradox/typed/actors.md

```diff
@@ -9,21 +9,39 @@ For the Akka Classic documentation of this feature see @ref:[Classic Actors](../
 
 ## Dependency
 
-To use Akka Actor Typed, you must add the following dependency in your project:
+To use Akka Actors, add the following dependency in your project:
 
 @@dependency[sbt,Maven,Gradle] {
   group=com.typesafe.akka
   artifact=akka-actor-typed_$scala.binary_version$
   version=$akka.version$
 }
 
+## Akka Actors
+
+The [Actor Model](http://en.wikipedia.org/wiki/Actor_model) provides a higher level of abstraction for writing concurrent
+and distributed systems. It alleviates the developer from having to deal with
+explicit locking and thread management, making it easier to write correct
+concurrent and parallel systems. Actors were defined in the 1973 paper by Carl
+Hewitt but have been popularized by the Erlang language, and used for example at
+Ericsson with great success to build highly concurrent and reliable telecom
+systems.
+
+The API of Akka’s Actors is similar to Scala Actors which has borrowed some of
+its syntax from Erlang.
+
+Since Akka enforces parental supervision every actor is supervised and
```

Comment:
Typed supervision is not really parental any more so we need to re-work this setence

Reply:
I think it could be good to link to some of general pages here. `general/actor-systems.md` and `general/actors.md` might be enough. We shouldn't introduce too many concepts at once. We can skip supervision here as far as I see it.

Reply:
@patriknw farther down in this very sentence the docs already have them:

```
Since Akka enforces parental supervision every actor is supervised and
(potentially) the supervisor of its children, it is advisable that you
familiarize yourself with @ref:[Actor Systems](../general/actor-systems.md) and @ref:[supervision](../general/supervision.md)
and it may also help to read @ref:[Actor References, Paths and Addresses](../general/addressing.md).
```

Reply:
does it? only in classic `/actors.md`, not in the `typed/actors.md`